indico/modules/events/surveys/templates/display/survey_list.html

```diff
@@ -30,24 +30,29 @@
                         {% if was_survey_submitted(survey) %}
                             <a class="i-button highlight disabled"
                                title="{% trans %}Survey already filled out{% endtrans %}">
-                                {% trans %}Fill out{% endtrans %}
+                                {%- trans %}Fill out{% endtrans -%}
                             </a>
                         {% elif not survey.is_active %}
                             <a class="i-button highlight disabled"
                                title="{% trans %}Survey is closed{% endtrans %}">
-                                {% trans %}Fill out{% endtrans %}
+                                {%- trans %}Fill out{% endtrans -%}
+                            </a>
+                        {% elif is_submission_in_progress(survey) %}
```

Comment:
Updated in https://github.com/indico/indico/pull/2873/commits/583143f2e1062cc0906e01458886738c759e0581